web/src/views/DandisetLandingView/DandisetMain.vue

```diff
@@ -197,6 +259,19 @@ export default {
   computed: {
     loggedIn,
     user,
+    contributors() {
+      const authors = _.map(this.meta.contributor, (author) => {
+        let affiliations = '';
+        if (author.schemaKey !== 'Organization' && author.includeInCitation) {
+          if (!_.isEmpty(author.affiliation)) {
+            affiliations = _.map(author.affiliation, (a) => a.name);
+            affiliations = affiliations.join(', ');
+          }
+          return { name: author.name, identifier: `https://orcid.org/${author.identifier}`, affiliation: affiliations };
+        } return {};
```

Comment:
For legibility, I would suggest:

```suggestion
        }
        
        return {};
```

Even better would be to maintain a return value variable that is modified within the if-block, but this is not a big deal.